fe/fe-core/src/main/java/com/starrocks/load/streamload/StreamLoadTask.java

```diff
@@ -1388,4 +1395,54 @@ public static StreamLoadTask read(DataInput in) throws IOException {
         task.setType(Type.PARALLEL_STREAM_LOAD);
         return task;
     }
+
+    public TStreamLoadInfo toThrift() {
+        readLock();
+        try {
+            TStreamLoadInfo info = new TStreamLoadInfo();
+            info.setLabel(label);
+            info.setId(id);
+            info.setLoad_id(loadId.toString());
```

Comment:
You can use `uuid` to instead of `TUniqueId(hi:4128505410267912933, lo:5661363850373276544)`

Reply:
load_id use `print_id()`

Reply:
@chaoyli @meegoo `load_id` is now updated